notebook/static/notebook/js/kernelselector.js

```diff
@@ -77,6 +78,15 @@ define([
                         .text(ks.spec.display_name)
                 )
             );
+
+            change_select_kernel_menu.append(
+                $("<option>").attr("id", "kernel-selectmenu-"+ks.name)
+                    .click( function () {
+                        that.set_kernel(ks.name);
+                    })
+                    .text(ks.spec.display_name)
+            )
```

Comment:
one liner comment on what exactly this piece of code is doing!

Reply:
Added comment explaining its function.